### PR TITLE
Feature/style upgrades

### PR DESCRIPTION
**Pull request by kpqdap**
This pull request includes several changes mainly focusing on improving the user interface and updating dependencies. The most significant changes include updating the `APP_VERSION` in `src/app-global.ts`, refactoring `handleSelectTenant` and `handleSelectUser` functions in `src/app/settings/admin/selectors.tsx`, and removing unnecessary files such as `src/app/settings/details/layout.tsx`, `src/app/settings/details/loading.tsx`, and `src/app/settings/details/page.tsx`.

App version update:

* [`src/app-global.ts`](diffhunk://#diff-6f6b4494ef9199e92a8afda39fbb184b4e854549a8dd44d0706e132a2b8fc3f2L1-R1): Updated the `APP_VERSION` from "1.6.1" to "1.6.2".

UI improvements:

* [`src/app/settings/admin/selectors.tsx`](diffhunk://#diff-d8c7a0319bdb53bcc2e684ece735ca738aa7582da4020f753306e673fbb441c8L19-R73): Refactored `handleSelectTenant` and `handleSelectUser` functions to improve user experience and readability. The functions now take a string value instead of an event, simplifying the code.
* [`src/features/ui/select.tsx`](diffhunk://#diff-60e284006943a508e738bbbc2f548d96c93d7117d2bf0c83444842ba102c576dL1-R95): Updated the `Select` component to use `@radix-ui/react-select` for a better user interface.
* [`src/features/ui/seperator.tsx`](diffhunk://#diff-c788f90a22afc5c51210b04b66ea754c81471e639e45ea6420906817d077b6a2R1-R68): Added a new `Separator` component using `@radix-ui/react-separator` for improved UI.

Removed unnecessary files:

* [`src/app/settings/details/layout.tsx`](diffhunk://#diff-a71aabc023ca3253f35d5a5968dc19337063473233cbdaeb3294559b681a1714L1-L5): Removed this file as it was no longer necessary.
* [`src/app/settings/details/loading.tsx`](diffhunk://#diff-d9affc18f16938ab55b5e7de536dcfe4378b7799166cc0463f580446df06c6e3L1-L5): Removed this file as it was no longer necessary.
* [`src/app/settings/details/page.tsx`](diffhunk://#diff-0793358f06aa0abddb5fae51af85bedc333393dd38e999870b5805418c0fa493L1-L30): Removed this file as it was no longer necessary.

Dependency updates:

* [`src/package-lock.json`](diffhunk://#diff-998cefe66c8f4a3c44a608be658fb845884ddbff34d948ca6c751f57186054b7L3-R9): Updated several dependencies including `applicationinsights`, `dompurify`, `highlight.js`, `marked`, `tailwind-merge`, and `@types/node`. [[1]](diffhunk://#diff-998cefe66c8f4a3c44a608be658fb845884ddbff34d948ca6c751f57186054b7L3-R9) [[2]](diffhunk://#diff-998cefe66c8f4a3c44a608be658fb845884ddbff34d948ca6c751f57186054b7R35-R36) [[3]](diffhunk://#diff-998cefe66c8f4a3c44a608be658fb845884ddbff34d948ca6c751f57186054b7L43-R56) [[4]](diffhunk://#diff-998cefe66c8f4a3c44a608be658fb845884ddbff34d948ca6c751f57186054b7L68-R70) [[5]](diffhunk://#diff-998cefe66c8f4a3c44a608be658fb845884ddbff34d948ca6c751f57186054b7L80-R82) [[6]](diffhunk://#diff-998cefe66c8f4a3c44a608be658fb845884ddbff34d948ca6c751f57186054b7R4836-R4841)

### CODE REVIEW

File: src/features/ui/select.tsx [hunk 1]
```diff
@@ -1,38 +1,97 @@
...
-import { cn } from "@/lib/utils"
+import Typography from "@/components/typography" // adjust import based on your file structure
+import { cn } from "@/lib/utils" // adjust import based on your file structure
 
-export interface SelectProps extends React.SelectHTMLAttributes<HTMLSelectElement> {
+export interface SelectProps {
   label?: string
   preventEmpty?: boolean
-  options: { value: string; label: string }[]
+  groups: { label: string; options: { value: string; label: string }[] }[]
```
Comment (Fyvel) [inline src/features/ui/select.tsx:11]: issue: we should reconsider before changing a generic Select UI component

File: src/features/ui/seperator.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import * as SeparatorPrimitive from "@radix-ui/react-separator"
+import React, { memo } from "react"
+
+import Typography from "@/components/typography"
+
+interface SeparatorProps {
+  title: string
+  description: string
+  linkSets: string[][]
+  titleId: string // Required for accessibility
```
Comment (Fyvel) [inline src/features/ui/seperator.tsx:10]: chore: remove AI generated comments ;)

File: src/features/ui/select.tsx (hunk 1 above)
Comment (QDAP-Fred) [reply src/features/ui/select.tsx:11]: issue: we should reconsider before changing a generic Select UI component

File: src/features/ui/seperator.tsx (hunk 2 above)
Comment (QDAP-Fred) [reply src/features/ui/seperator.tsx:10]: chore: remove AI generated comments ;)

**Review by qdap-bp — APPROVED** [review]

**Pull request merged**